### PR TITLE
[sgx-untrusted-alloc] Add support for infinite capacity of untrusted memory

### PR DESCRIPTION
**Pull request by jessehui**

### CODE REVIEW

**Comment by volcano0dr** [comment]
The current implementation can cause infinite expansion of untrusted memory. I think we should consider implementing chunk recovery.

File: src/libos/crates/sgx-untrusted-alloc/src/untrusted_allocator/mod.rs [hunk 1]
```diff
@@ -13,86 +13,99 @@ const PAGE_SIZE: usize = 4096;
 // The address of a block returned by malloc or realloc in GNU systems is always a multiple of eight (or sixteen on 64-bit systems).
 const DEFAULT_ALIGNMENT: usize = 16;
 
+const DEFAULT_CHUNK_SIZE: usize = 4 * 1024; // FOR TEST ONLY
```
Comment (volcano0dr) [inline src/libos/crates/sgx-untrusted-alloc/src/untrusted_allocator/mod.rs:16]: Please modify it to 16M.

Comment (jessehui) [reply src/libos/crates/sgx-untrusted-alloc/src/untrusted_allocator/mod.rs]: Done. Thanks for you review.

**Review by volcano0dr — APPROVED** [review]
LGTM

**Pull request merged**